string first_number
string sign
string second_number